client/src/components/survey/SurveyPage1.jsx

```diff
@@ -106,6 +107,17 @@ const SurveyPage1 = props => {
     setSurveyPage(surveyPage + 1);
   };
 
+  // HOOKS AND METHODS for PopUpContactAlert
+  const [open, setOpen] = React.useState(false);
+
+  const handleClickOpen = () => {
+    setOpen(true);
+  };
+
+  const handleClose = () => {
+    setOpen(false);
+  };
+
```

Comment:
I would suggest moving `open` (named more precisely) into Redux set, with an accompanying function to update that value. That way, we don't have pass around functions as props.

Even if not, we don't need separate handlers here, and can just call `setOpen(true)` or `setOpen(false)` directly.